bin/plugin/config.js

```diff
@@ -3,20 +3,20 @@ const gitRepoOwner = 'WordPress';
 /**
  * @typedef WPPluginCLIConfig
  *
- * @property {string} slug Slug.
- * @property {string} name Name.
- * @property {string} team Github Team Name.
- * @property {string} versionMilestoneFormat printf template for milestone
+ * @property {string} slug                    Slug.
+ * @property {string} name                    Name.
+ * @property {string} team                    Github Team Name.
+ * @property {string} versionMilestoneFormat  printf template for milestone
  *                                           version name. Expected to be called
  *                                           with a merged object of the config
  *                                           and semver-parsed version parts.
```

Comment:
```suggestion
 * @property {string} versionMilestoneFormat  printf template for milestone
 *                                            version name. Expected to be called
 *                                            with a merged object of the config
 *                                            and semver-parsed version parts.
```

Identation of the subsequent lines should align with the start of the first comment